Aspire.Hosting.* READMEs (#3828)
microsoft/aspire: src/Aspire.Hosting/README.md

# Aspire.Hosting library

> This is the core package for .NET Aspire. It provides the building blocks for the distributed application

Reviewer: ```suggestion
Core abstractions for the .NET Aspire application model. It provides the building blocks for the distributed application
```

hosting model. This package should not be referenced by AppHost projects directly. Instead use the `Aspire.Hosting.AppHost`
package to add a transitive referencing including custom build targets to support code generation of metadata

> types for referenced .NET projects.

Author: No we should mention it ... good call.
